### ISSUE TITLE
NTR: knee joint effusion

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#10622]
**Preferred term label:**
knee joint effusion

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 31575400. Abnormal accumulation of fluid within the knee joint or around the knee. Symptoms maye include, pain, swelling, and stiffness. This increased intra-articular amount of fluid (= synovial fluid) is not specific of a particular disorder, as can occur in a variety of settings but draws attention to the presence of a joint disorder, such as knee osteoarthritis, or rheumatoid arthritis

**Parent term (use hpo.jax.org/app)**
Abnormal knee morphology HP:0034669

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-0113-912X, 0000-0002-7371-8158

**Comment by pnrobinson** [comment]
@GrieseAG 

Adding new term: 
- ID: HP:6001020
- label: Knee joint effusion
- parents: [Abnormal knee morphology (HP:0034669)](https://hpo.jax.org/app/browse/term/HP:0034669)
- pmid: PMID:31575400
- def: Abnormal accumulation of fluid within the knee joint or around the knee. Symptoms maye include, pain, swelling, and stiffness.
- comment: Increased intra-articular amount of (synovial) fluid is not specific for a particular disorder, as can occur in a variety of settings but draws attention to the presence of a joint disorder, such as knee osteoarthritis, or rheumatoid arthritis.
- orcid: orcid:0000-0002-7371-8158
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10622
- synonyms:
